src/main/java/com/reimbursements/reporitories/StatusRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.reimbursements.reporitories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.reimbursements.models.Status;
+
+
+
+public interface StatusRepository extends JpaRepository<Status, Integer> {
+	public Status getByStatusID(int statusID);

```

Comment:
This method is not required. By default, `JpaRepository` implements some basic methods: `get` by ID is one of them.